src/site/content/en/blog/preload-responsive-images/index.md

```diff
@@ -0,0 +1,201 @@
+
+To answer that I created two copies of a [demo PWA shop](https://github.com/GoogleChromeLabs/sample-pie-shop): [one that does not preload images](https://20190710t144416-dot-pie-shop-app.appspot.com/apparel), and [one that preloads some of them](https://20190710t132936-dot-pie-shop-app.appspot.com/apparel). Since the site lazy loads images using JavaScript, it's likely to benefit from preloading the ones that will be in the initial viewport.
+
+That gave me the following results for [no preload](https://www.webpagetest.org/result/190710_VM_30b9d4c993a1e60befba17e1261ba1ca/) and for [image preload](https://www.webpagetest.org/result/190710_7B_a99e792121760f81a270b4b9c847797b/). Looking at the raw numbers we see that [Start Render](https://sites.google.com/a/webpagetest.org/docs/using-webpagetest/quick-start-quide#TOC-Start-Render:) stayed the same, [Speed Index](/speed-index/) slightly improved (273 ms, as images arrive faster, but don't take up a huge chunk of the pixel area), but the real metric which captures the difference is the [Last Painted Hero](https://github.com/WPO-Foundation/webpagetest/blob/master/docs/Metrics/HeroElements.md) metric, which improved by 1.2 seconds. 🎉🎉
+
+Of course, nothing captures the visual difference quite like a filmstrip comparison:
+<figure class="w-figure">
+<img src="./example-3.png" alt="Screenshot of WebPageTest filmstrip comparison showing preloaded images are displayed about 1.5 seconds faster.">
+<figcaption class="w-figcaption">The filmstrip shows that images arrive significantly faster when preloaded, resulting in a hugely-improved user experience.</figcaption>
+<figure> 
+
+## Preload and `<picture>`?
+
+If you're familiar with responsive images, you may be wondering "What about [`<picture>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture)?".
+
+We're talking about adding a preload equivalent for `srcset` and `sizes`, but not the `<picture>` element, which tackles the ["art direction"](/codelab-art-direction/) use-case.
+
+Why is this use-case being "neglected"?
+
+While there's interest in solving that use case as well, there are still a number of [technical issues to sort out](https://calendar.perfplanet.com/2018/how-the-sausage-is-made-webperfwg-meeting-summary/) which means that a solution here would have significant complexity. On top of that, it seems like for the most part, the use-case can be addressed today, even if in a hacky way (see below). 
+
+Given that, the Web Performance WG decided to ship `srcset` first and see if the demand for equivalent `picture` support arises.
+
+If you do find yourself in a position to preload `<picture>` you may be able to use the following technique as a workaround.
+
+Given the following scenario:
+
+```html
+<picture>
+    <source src="small_cat.jpg" media="(max-width: 400px)">
+
+    <source src="medium_cat.jpg" media="(max-width: 800px)">
+
+    <img src="huge_cat.jpg">
+</picture>
+```
```

Comment:
Please remove the extra spacing

![image](https://user-images.githubusercontent.com/4713486/65641116-cab2fd00-dfa0-11e9-8535-e316d9852948.png)